BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/ui/theme/Type.kt

```diff
@@ -49,35 +50,60 @@ private val fontFamilyLato = FontFamily(
 )
 
 val typography = Typography(
-    defaultFontFamily = fontFamilyLato,
-    h1 = TextStyle(
+    displayLarge = TextStyle(
         fontFamily = fontFamilyKulim,
         fontWeight = FontWeight.Light,
-        fontSize = 28.sp,
-        letterSpacing = (1.15).sp
+        fontSize = 57.sp,
+        lineHeight = 64.sp,
+        letterSpacing = (-0.25).sp
     ),
-    h2 = TextStyle(
+    displayMedium = TextStyle(
         fontFamily = fontFamilyKulim,
-        fontSize = 15.sp,
-        letterSpacing = (1.15).sp
+        fontSize = 45.sp,
+        lineHeight = 52.sp
     ),
-    h3 = TextStyle(
-        fontWeight = FontWeight.Bold,
-        fontSize = 14.sp,
-        letterSpacing = 0.sp
+    displaySmall = TextStyle(
+        fontFamily = fontFamilyKulim,
+        fontSize = 36.sp,
+        lineHeight = 44.sp
     ),
-    body1 = TextStyle(
-        fontSize = 14.sp,
-        letterSpacing = 0.sp
+    titleMedium = TextStyle(
+        fontFamily = fontFamilyLato,
+        fontWeight = FontWeight(500),
+        fontSize = 16.sp,
+        lineHeight = 24.sp,
+        letterSpacing = (0.15).sp
+    ),
+    bodySmall = TextStyle(
+        fontFamily = fontFamilyLato,
+        fontSize = 12.sp,
+        lineHeight = 16.sp,
+        letterSpacing = (0.4).sp
     ),
-    button = TextStyle(
-        fontWeight = FontWeight.Bold,
+    bodyMedium = TextStyle(
```

Comment:
It looks like some of these texts are bold when I check the designs. Can you double check?

Reply:
the bolding is done through the font weight in these I believe!